app/javascript/controllers/fly_out_controller.js

```diff
@@ -1,20 +1,45 @@
 import { Controller } from 'stimulus'
 
 export default class extends Controller {
-  static targets = [ 'sidePanel', 'programSession' ]
-  static values = { revealedElement: String }
+  static targets = [ 'sidePanel', 'programSession', 'filterWrapper' ]
```

Comment:
I wonder if `programSession` could be a more generic named target like `listItem` so that it could be reused in other future situations?

Reply:
Thanks for noting this. It was a misstep for sure.
I like `listItem` as well as anything else, and that is what I updated to. 

I tried to correct a couple of other instance, where I had made a 'less than clear' naming choice. Examples being fixing the schedule to make it clear that the views are using `time_slots` and not `program_session`, and some instances were I was misnaming a `session_format_config` as a `session_format`. 

Naming. Always so hard.